frontend/src/queries/nodes/DataTable/renderColumn.tsx

```diff
@@ -182,6 +182,9 @@ export function renderColumn(
                 {String(value)}
             </CopyToClipboardInline>
         )
+    } else if (key.startsWith('user.') && isTimeToSeeDataSessionsQuery(query.source)) {
+        const [parent, child] = key.split('.')
+        return typeof record === 'object' ? record[parent][child] : 'unknown'
```

Comment:
General comment for this file, for sure out of scope here: This `renderColumn` is getting rather large. We should probably split it up by query type.